### PR TITLE
OSDOCS-5730 linking to vmware NSX docs

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.9+

Issue:
https://issues.redhat.com/browse/OSDOCS-5730

Link to docs preview:
https://58603--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned-customizations

QE review:
- [X] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@sgaoshang @racedo PTAL at this change for the vmware NSX compatibility table. Thank you

File: modules/installation-vsphere-infrastructure.adoc [hunk 1]
```diff
@@ -78,7 +78,7 @@ You can host the VMware vSphere infrastructure on-premise or on a VMware Cloud V
 ifndef::vmc[]
 |Optional: Networking (NSX-T)
 |vSphere 7.0 Update 2 and later
-|vSphere 7.0 Update 2 is required for {product-title}. VMware's NSX Container Plugin (NCP) is certified with {product-title} 4.6 and NSX-T 3.x+.
+|vSphere 7.0 Update 2 is required for {product-title}. For more information about the compatibility of NSX and {product-title}, see VMware's link:https://docs.vmware.com/en/VMware-NSX-Container-Plugin/index.html[NSX container plugin documentation].
 endif::vmc[]
```
Comment (racedo) [inline modules/installation-vsphere-infrastructure.adoc:82]: I'd mention that it's in the release notes section

**Review by snarayan-redhat — COMMENTED** [review]
LGTM

**Comment by racedo** [comment]
LGTM

**Pull request merged**